### ISSUE TITLE
Position of value in value-plotters

### ISSUE THREAD

**Issue by interior-night** [frauzufall/ofxGuiExtended#38]
Having the value on the left hand side, followed by the plotter's name, makes the name bounce around a little too much when not using a monospaced font. It might look better if the name was on the left and the values on the right, as is currently the case with sliders. And, down the track, it would be great if the first digit of the value could remain static. For that, and to deal with non-monospaced fonts, I suppose either each digit should be artificially monospaced (if that's do-able?) otherwise the overall value should be left-aligned to a container (taking into account the fact that negative values add a character).

**Comment by frauzufall** [comment]
Yes I will change this layout, I also saw this in your GUI, the current solution is good for the FPS plotter but for most other cases its not the best solution.
